ts/receiver/dataMessage.ts

```diff
@@ -104,8 +103,21 @@ export function isMessageEmpty(message: SignalService.DataMessage) {
   );
 }
 
-function isBodyEmpty(body: string) {
-  return isEmpty(body);
+/**
```

Comment:
Isn't this more about ignoring reactions when checking if a message is empty not "no" reactions?

Reply:
updated the comments